### DISCUSSION TITLE
节点比较近时的连线路由问题

### DISCUSSION THREAD

**Discussion by luoxy1001** [antvis/X6#2162] [Q&A]
节点比较近的时候连线就会变更方向（如左边的ecu1节点），请问这个节点之间的距离多少的时候会这样？有地方设置这个距离吗？
注：can1上两边的方形不是链接桩，下面两个ecu上面的小矩形是链接桩。
我的连线路由配置：
![image](https://user-images.githubusercontent.com/19257328/171089001-b063f8ce-d04c-4be5-93d2-7cd76a735e64.png)

**Comment by NewByVector** [comment]
你把 `excludeTerminals` 去掉试试呢

**Reply by luoxy1001** [reply]
去掉也不行的，我就是觉得有这个问题才加上的excludeTerminals

**Reply by NewByVector** [reply]
有 demo 可以看吗，可以整理一个 codesanbox 复现下，这样会比较好定位问题。

**Reply by luoxy1001** [reply]
帮忙看看：https://codesandbox.io/s/gracious-liskov-c8xenq
